coderd/database/migrations/000240_notification_workspace_updated_version_message.down.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE notification_templates
+SET body_template = E'Hi {{.UserName}}\n' ||
+                    E'Your workspace **{{.Labels.name}}** has been updated automatically to the latest template version ({{.Labels.template_version_name}}).'
+WHERE id = 'c34a0c09-0704-4cac-bd1c-0c0146811c2b';
```

Comment:
nit: Add a comment describing this magic UUID :)

Reply:
Good call-out, however there's a lot of precedent in other migrations of not adding comments, I'm afraid.
These UUIDs are "well-known" in that they're searchable in the codebase, at least.